lib/sqs/index.js

```diff
@@ -11,10 +11,14 @@ const compress = util.promisify(zlib.gzip);
 const decompress = util.promisify(zlib.gunzip);
 const { computeMessageSize, promisify, setHttpAgent } = require('../utils');
 
-// 256kb => b
+// 256kb => b: (256kb * 1024 bytes =  262144)
 const MAX_MSG_SIZE = 262144;
-//64kb => bytes
-const MAX_COMPRESS_MSG_SIZE = 65536;
+/*
+ * Max size of message that does not need to be compressed as it can be sent via 1 SQS Operation
+ *  64kb => b: (64kb * 1024 bytes = 65536 bytes - 50 bytes = 65486)
+ *  -50 bytes to compensate for size increase after compressing with no compression level
```

Comment:
how did we settle on 50 bytes for this buffer?

Reply:
Series of test up to 64kb never saw more than 40 bytes added post compression with no compression rate.  Added another 10 byte buffer for good measure.